src/BackTranslationHelper/BackTranslationHelperCtrl.cs

```diff
@@ -1084,7 +1118,13 @@ private void ButtonPinToTop_Click(object sender, EventArgs e)
                                     ? global::BackTranslationHelper.Properties.Resources.pindown
                                     : global::BackTranslationHelper.Properties.Resources.pinup;
 
-            if ((Parent != null) && (Parent is Form parentForm))
+			// Now that I'm using a TableLayoutPanel in the Ptx plugin form, it may not be the parent.
+			//  So go up the parent chain until we find a Form
+			var parent = (Control)this;
```

Comment:
```while(parent?.Parent != null && !parent is Form)
      parent = parent.Parent;
   if(parent is Form)
      parent.TopMost = newCheckState;
   ```
I suggest this alternative, fewer parenthesis, same behavior I think. 'Is' returns false on a null

Reply:
done